### ISSUE TITLE
Has This Project Been Abandoned; Can Anyone Help? 

### ISSUE THREAD

**Issue by powline99** [tomlandia/fuckr#57]
@tomlandia @victorgrego @shyrat @genevera @Kav28

Hey Tom, Victor, Shyrat and Genevera, **anyone who can help:**

Has this project  been abandoned?? I know Tom is busy but he said many things needed to be updated with nw 0.12.1 . Can anyone else update or help to fix the issues. It's like mine never logs in anymore, I tried blocking on the software and they are still there on the phone's app. Also messages only appear in Fuckr but not in grindr chat when someone messages me and I close out. The process does not get killed when the program is closed until a hard reset. IT would be nice to have the ability to do multiple logins, say if someone blocks you to check using a different login / logins for different moods / times. The program retains the last login info even if you logout, especially if there is a memory / program crash.

Thanks for ALL YOUR WORK!
;-)

____________________________________________
@tomlandia
tomlandia commented on May 7 • edited
I'm confused as if it's yet another weird v1.6.1 bug but going to implement XMPP disconnect just to make sure.

@genevera
 genevera commented on May 27
@tomlandia i think maybe it's confirmchatmessagesdelivered that deletes them from the server. I've been experimenting with that locally and it seems to be the case.

_______________________________________

powline99 commented on May 3 • edited
@tomlandia Fuckr will not logout, it stays in the background processes after the program is closed or crashes (seems I am still on Grindr still). Multiple instances taking up resources. Have to do full shutdown to make them go away.
 @tomlandia
 Owner
tomlandia commented on May 7
Yeah v1.6.1 is a shitshow sorry. It's not the one on Fuckr.me. Going to release 1.6.2 using nw 0.12.1.

_______________________________________
nw.js ›
Single process mode is now supported in 0.14.x

_Apr 20
You can launch nw with command line argument '--single-process' from upstream. And then nw will run in single process mode.

**Comment by genevera** [comment]
Hi @powline99 - have you pulled the latest version of master / 1.6.1 and built the application therefrom?  If you're on OSX I can give you the one-liner I use to build and launch the app each time.  I haven't been doing much beyond playing around with increasing the accuracy of long distance pinpointing and disallowing the confirmChatMessagesDelivered API call.  I may look into improving a couple of missing features / bugs that I've encountered in my own use but seeing as I am not the author of this project do not take that as evidence that it hasn't been dropped :)

**Comment by powline99** [comment]
Hi @genevera thank you for taking your time to help, unfortunately I am on Windows 7 32bit. The biggest annoyance to me is it's like Fuckr is running a separate profile using my username and password for my phone's app. I get a error like icon and then I see messages that do not show up while I do not use the software in my app. Before it shared the same profile. It's really weird that since the login and password across platforms does not share the profile. Perhaps it is not connecting to the server and by me blocking on the pc program it's not even communicating with gindr (When I open the iphone app all the blocks are still there, before we would get a eclipse server error and it would log you out of the software / shutdown). Kinda like if you never login to grindr. Secondly it is very annoying that when the software crashes on my pc, it has a phantom process and when I open it up again it continues to run and use resources in the background, it is not killing the process and the only way to get rid of the stacks of them is to restart the computer  (when it closes everything should be killed and not stay logged in). I think tom found some Google single process work around below. 

Thanks for all your help!
P

@tomlandia was working on these issues: 
_________________________________________________________________

Error Pops up every so often (while chatting, deleting, or blocking), eclipse wants login (I must go put in CAPTCHA phrase and click to get rid of Fuckr errors. Shows brief screen with unblocked local profiles. Second image is a error type thing after CAPTCHA phrase and then it goes to main screen.
2016-04-28_082528

2016-04-28_072714

Could we implement that "Looks like https://groups.google.com/forum/#!topic/nwjs-general/LtJql3LSV44 may offer a solution provided it's stable and we can add a nw command line flag to releases"
__________________________________________________________________

powline99 commented on May 3 • edited
@tomlandia Fuckr will not logout, it stays in the background processes after the program is closed or crashes (seems I am still on Grindr still). Multiple instances taking up resources. Have to do full shutdown to make them go away.
 @tomlandia
 Owner
tomlandia commented on May 7
Yeah v1.6.1 is a shitshow sorry. It's not the one on Fuckr.me. Going to release 1.6.2 using nw 0.12.1.

_________________________________________________________________

@tomlandia Map function not working when I click on world icon in profiles, it just spins and does nothing. Map does work if you use drop down view in main screen. Could this be my spam blocker in Chrome or setting not to open new windows, just tabs?

Thanks!
 @Kav28
 
 Kav28 commented on Apr 28
Have had that prob but think it happens sporadically - might be due to
load.
…
@tomlandia
 Owner
tomlandia commented on May 7
V1.6.1 is a shitshow sorry. It's not the one on Fuckr.me. Going to release 1.6.2 using nw 0.12.1.

______________________________________

@tomlandia 
Can't get the Conversion Metric / US Setting Drop Down to Work. I logged in and out. Tried to close down the program / open. Nothing seems to change not even my profile settings to inches and pounds ;-p
 @tomlandia
 Owner
tomlandia commented on May 7
Works fine in dev but only in profile view.

__________________________________________

This one is a bit strange. I start chats while at my laptop on the same username as my IOS mobile. I know chats do not save across platforms but before I was able to start a convo and then go mobile and also have a new chat open up if the person responded. Now the responses only go to Fuckr even if the software is closed and I am fully mobile and logged in for a few hrs??? I wonder if this has to do with the phantom processes / sessions after crash issue I posted before, it still thinks I am logged in and so never truly closes software / logs out. I tried to manually log out same thing, but if a session is running after a crash and stuck in memory then I suppose it does nothing.
 @Kav28
 
 Kav28 commented on Apr 28
If I start a chat with the most current version of Fuckr, sign out and then
start a older version of Fuckr replies to the chat will not come in until I
sign out and then start the new version of Fuckr. If I have two users (not
used at the same time) chat history of both users will be visible but new
messages will not appear until I am signed in as the user intended for the
chat (changed behavior from earlier versions).
…
@tomlandia
 Owner
tomlandia commented on May 7 • edited
I'm confused as if it's yet another weird v1.6.1 bug but going to implement XMPP disconnect just to make sure.
@genevera
 
genevera commented on May 27
@tomlandia i think maybe it's confirmchatmessagesdelivered that deletes them from the server. I've been experimenting with that locally and it seems to be the case.

**Comment by newtofuckr** [comment]
i have also wondered if the program has been abandoned. i had never used fuckr/grindr before until February, and used it regularly once I got it. i had occasional msg blackouts, but for the most part it worked well. however the new version made things worse... ive only gotten a message from one user in the past 40 days, and i asked if he was real and not spam and to type a word to prove he was real, and he did.

i am able to log online, but im never visible to other members, and never get email from other members (99% of the time). this version for some reason, works a lot worse than the version before it. i installed the earlier version after all the problems with the new version, and there's no improvement, i dont know why. i have both versions on my computer, currently.

i do want to comment that i do like the search options on the new version of age, ethnic background, types (i love otter types and Middle Eastern types for example, and it brings up so many attractive guys!). however this new version doesnt give us the option to choose our own "tribes" so that we show up for other members, for example i cant select myself as an otter. 

also, in both versions, i have never been able to view my own profile. is it possible in a future version that we can view our own profile? 

i have been checking for a new version since mid may when all the problems started, and ive pretty much given up on grindr/fuckr, and been telling people on other sites, that i never can use grindr anymore. its really frustrating since i see so many other guys are actually on grindr and i cant even communicate with them. 

i get this msg pop up a lot on the newer version:
**"XMPP chat error. If you're using public wifi, XMPP protocol is probably blocked."**

I would have attached an image I screenshotted when this happened, but it said that github doesnt allow bitmap files to be uploaded for some reason, as that is what i saved it as, as that is the default format when one screenshots into the paint program. 

I suspect the above reason is why fuckr never will show me online/get msgs anymore. ive also tried using a different email account under registration to try to see if that will allow me on grindr/fuckr and it has no impact. 

If there is a way to have XMPP protocol UNBLOCKED someone tell me how, and I will try to see if I can do that. the only internet connection i have is wireless.

**Comment by newtofuckr** [comment]
im chatting to a guy on another site that says he can get grindr to work fine today (June 6, 2016)... but im assuming he's using the real grindr, and not fuckr. i never knew of anyone i chatted with on there that was using fuckr other than myself. 

theres a few reasons i dont use grindr on my phone.
a) ive never been an app person in general. about the only app i use is my photo album, to take pictures with and send pictures out. 
b) file size. even when fuckr worked right back in february/march and part of april, when conversations kept going on and on, it uses a lot of memory not just to operate the program, but in file sizes of conversation logs. i dont want all that weighing down my phone. a computer can handle all that far easier. also, a computer has an expanded window, so you dont have everything compressed to a small screen. 

i tried using bluestacks for another program once, but i got rid of it. im not a fan of google at all, and i dont like how they pervade nearly every device and website possible, them admitting to spying and storing private data on everyone, just isnt something i condone. i try to avoid anything that deals with google. however if fuckr incorporates technology that is google-based, im willing to accept that, but ill still avoid google in general otherwise. 

maybe someone could track down the creator of "Chat Client" that was used for gay dot com back in the 2000's era, and get some technical advice from them, and maybe even see if they want to contribute to this project. one of the reasons gay dot com was so successful was because of chat client, and once gay dot com changed the design layout of their entire website, and blocked chat client from accessing them, nearly half of all members of their site (including me) left. i probably used gay dot com about 300 days out of 365 days for many years back during their peak era. ive only used gay dot com about 7 days in the past 2 years, now. they shot their own foot off, they had such an empire of members (i met guys from it), but i digress. 

im not technically advanced when it comes to technology about the builds on here. i tried doing something back in may (you contributed to that discussion), but the program wouldnt edit at all for me. so i gave up. 

as for removing all traces... i did at first, and this caused more problems than less. because when i got the newer version, it would not create its own folder, like the older version had. i had removed too much traces of the older version. i have considered removing all traces of both versions i have now, and then downloading the latest version again, and see if that works. but if i do that, ill erase all the msg histories currently stored. 

grindr is good at bringing in so many attractive guys, which is why its so frustrating how i cant interact with them. its like being a kid in a candy store and not having any teeth. :D 

i do hope that tom, or someone else continues working on this, and lets the mantle of fuckr continue on in years to come, with improvements that are so profound that we forget these errors that are occurring. due to my ignorance of program technology, all i have to do is wait and wish more enlightened wisdom on the program creator(s).

**Comment by newtofuckr** [comment]
I ended up deleting all the data of fuckr from my computer both the recent version and the older version and downloaded the april version. I was able to get online briefly, but I was unable to update any of the age/weight/height information. I tried, and went to a different window, and when I went back to this page, they were all cleared. After re-entering it, it was cleared again.

and now I cannot get online at all. it says "requests to the server have been blocked by an extension". with a frowny face above it. i would post a screenshotted pic but github doesnt allow bmp files. 

does this mean that grindr has blocked all fuckr users from getting online?
if so, i hope tom can bypass this. 

right now, it is impossible to get online at all, and when i was actually online, i was able to view profiles, and now i cant even do that. it seems each time i remove fuckr from the registry, a downloaded version is even worse than the one before. 

if tom doesnt want to be part of this project anymore, maybe he could appoint someone else in this community to work on it?

**Comment by powline99** [comment]
@newtofuckr I don't know what to tell you, haha. Well just have to wait and see if their is a response. I would suggest you get the app for your phone in the mean time or download like bluestacks and install it.

**Comment by tomlandia** [comment]
Sorry about not updating sooner. I think I've fixed the recent chat problem even if Grindr now does certificate pinning so it's harder to reverse engineer (documentation will have to be updated). I'll release a new version tonight (European time) latest. Closing this issue as it's redundant with #61.
